apis/refs/v1beta1/computerefs.go

```diff
@@ -54,14 +44,7 @@ func ResolveComputeNetwork(ctx context.Context, reader client.Reader, src client
 		if ref.Name != "" {
 			return nil, fmt.Errorf("cannot specify both name and external on computenetwork reference")
 		}
-
-		tokens := strings.Split(ref.External, "/")
-		if len(tokens) == 5 && tokens[0] == "projects" && tokens[2] == "global" && tokens[3] == "networks" {
-			return &ComputeNetwork{
-				Project:          tokens[1],
-				ComputeNetworkID: tokens[4]}, nil
-		}
-		return nil, fmt.Errorf("format of computenetwork external=%q was not known (use projects/<projectId>/global/networks/<networkid>)", ref.External)
```

Comment:
Let's just change this to a warning.  We don't currently have a suitable context (like MapContext), so maybe just a klog or similar.